src/Chapter1.hs

```diff
@@ -652,9 +665,13 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
-
+firstDigit :: Int -> Int
+firstDigit n = firstDigitAbs (abs n)
+  where
+    firstDigitAbs :: Int -> Int
+    firstDigitAbs absN
+      | absN `div` 10 == 0 = absN
+      | otherwise = firstDigit (absN `div` 10)
```

Comment:
This is absolutely great idea to have the additional recursive function 🧠 
So you can just check on `abs` only in the beginning 👍🏼 
I guess, you meant this here:

```suggestion
      | otherwise = firstDigitAbs (absN `div` 10)
```